### PR TITLE
Notes about the sources for the compiler interface

### PR DESCRIPTION
**Pull request by Duhemm**

### CODE REVIEW

**Comment by eed3si9n** [comment]
Besides capitalization, looks good.

**Comment by Duhemm** [comment]
Just fixed the capitalization

File: src/reference/03-Developers-Guide/00.md [hunk 1]
```diff
@@ -38,6 +39,13 @@ The launch jar will look in its own classpath for a boot properties file.  The
 [sbt/sbt](https://github.com/sbt/sbt) project then pulls in the raw JAR and
 [injects the appropriate boot.properties files for sbt](launcher-inject).
 
+#### [compiler-interface][Compiler-Interface]
+
+The compiler interface is the part of sbt that is recompiled against the version of
+the Scala compiler that is used to compile your projects.
+
+It is in charge of extracting information from your source code by walking the compiled
+trees and is therefore very sensible to the Scala version in use.
```
Comment (dwijnand) [inline src/reference/03-Developers-Guide/00.md]: sensitive?

Comment (Duhemm) [inline src/reference/03-Developers-Guide/00.md]: @dwijnand French writer spotted 😮 Thanks!

Comment (eed3si9n) [inline src/reference/03-Developers-Guide/00.md]: I don't know if changing this to sensible make sense either.

**Pull request merged**